src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -656,7 +655,7 @@ private boolean checkForDuplicateAddressTypes(User user) {
     Future<Boolean> future = Future.future();
     Criterion criterion = new Criterion(
           new Criteria().addField(AddressTypeAPI.ID_FIELD_NAME).
-                  setJSONB(false).setOperation("=").setValue("'" + addressTypeId + "'"));
+                  setJSONB(false).setOperation("=").setVal(addressTypeId));
```

Comment:
Can you replace the postgresClient.get method by a PgUtil.getById invokation so that this duplicated code gets removed?